packages/roosterjs-editor-api/lib/format/cacheGetHeaderLevel.ts

```diff
@@ -4,6 +4,11 @@ import { PluginEvent } from 'roosterjs-editor-types';
 
 const EVENTDATACACHE_HEADER_LEVEL = 'HeaderLevel';
 
+/**
+ * Get the header level in current selection
+ * @param editor The editor instance
+ * @param event The plugin event
```

Comment:
Suggest to add comment that event is optional

Reply:
Changed.